{"name": "test_c004_03_error_field", "status": "passed", "steps": [{"name": "Logged_in user details, error field validation", "status": "passed", "start": 1631722809005, "stop": 1631722809005}], "start": 1631722809005, "stop": 1631722809005, "uuid": "985d736f-11fd-47b1-9666-1fa47e455688", "historyId": "85890164b9d61768265db0b872b55fdf", "testCaseId": "e7c9fcc9e2d235a18268dc52a70de04e", "fullName": "testcases.test_c006_current_logged_in_user_401#test_c004_03_error_field", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_c006_current_logged_in_user_401"}, {"name": "host", "value": "QupsAnik"}, {"name": "thread", "value": "18880-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_c006_current_logged_in_user_401"}]}